src/funcCoreTools/installFuncCoreTools.ts

```diff
@@ -20,10 +22,12 @@ export async function installFuncCoreTools(context: IActionContext, packageManag
     switch (packageManagers[0]) {
         case PackageManager.npm:
             const distTag: INpmDistTag = await getNpmDistTag(context, version);
+            lastCoreToolsInstallCommand = `npm install -g ${funcPackageName}@${distTag.tag}`;
```

Comment:
Can you make this an array of args, and then use a spread operation on the cpUtils.executeCommand call?

```
lastCoreToolsInstallCommand = ['npm', 'install', '-g', `${funcPackageName}@${distTag.tag}`]
await cpUtils.executeCommand(ext.outputChannel, undefined, ...lastCoreToolsInstallCommand);
```

I think that should work. And then for the copy command, you can just do `lastCoreToolsInstallCommand.join(' ');`

Added benefit of copying the `brew tap` command as well in case that is the one that errored out instead of `brew install`